src/main/java/org/openbase/bco/bcozy/view/BackgroundPane.java

```diff
@@ -61,6 +74,15 @@ public BackgroundPane(final ForegroundPane foregroundPane) throws InstantiationE
             this.editingLayerPane = new SimpleUnitSymbolsPane();
             this.editingLayerPane.setPickOnBounds(false);
 
+            // Pane layer for fast overview over power draw
+            FXMLLoader fxmlLoader = new FXMLLoader(getClass().getClassLoader().getResource(POWER_DRAW_PANE_FXML_LOCATION));
```

Comment:
Please use the [FXMLProcessor](https://github.com/openbase/jul/blob/master/visual/javafx/src/main/java/org/openbase/jul/visual/javafx/fxml/FXMLProcessor.java) like discussed instead loading the fxml manually. Refactoring of is not required within this pull request but a hint for further fxml handling.

Reply:
Will be refactored in future commits